pkgs/misc/cups/0001-TargetConditionals.patch

```diff
@@ -0,0 +1,29 @@
+From 1204c841999808ba27267a0039777dcbccdcd6e3 Mon Sep 17 00:00:00 2001
+From: toonn <toonn@toonn.io>
+Date: Sun, 27 Jun 2021 12:30:08 +0200
+Subject: [PATCH] TargetConditionals
+
+---
+ test/ippserver.c | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/test/ippserver.c b/test/ippserver.c
+index 38b304f..68ccab1 100644
+--- a/test/ippserver.c
++++ b/test/ippserver.c
+@@ -25,7 +25,11 @@
+  * Include necessary headers...
+  */
+ 
+-#include <config.h>			/* CUPS configuration header */
++#ifdef __APPLE__
++#  include <xcode/config.h>		/* CUPS configuration header macOS */
++#else
++#  include <config.h>			/* CUPS configuration header */
++#endif /* __APPLE__ */
```

Comment:
This should probably be upstreamed.

Wasn't created cups created originally for apple?

Reply:
Yes, the problem is that we don't use an Xcode build environment, which is probably a project assumption when building for macOS. So I'm not sure this is upstreamable.